src/app/containers/main/__snapshots__/index.test.jsx.snap

```diff
@@ -43,11 +47,26 @@ exports[`Main component renders properly 1`] = `
   <div
     className="attribution"
   >
-    <div>
-      <img
-        className="attribution"
-        src="test-file-stub"
-      />
+    <div
+      className="footer"
+    >
+      <div
+        className="main"
+      >
+        <div
+          className="title"
+        >
```

Comment:
This might make this the last time you ask me to review anything... but...

![my eyes](https://media0.giphy.com/media/IepbiTPt2dORa/giphy.gif?fingerprint=e1bb72ff59a2fc0f4737644f41fbb3f9)

Reply:
Snapshots are just generated code lol

Reply:
Oh! DIdn't notice that was a snapshot.